src/utils/guid.test.js

```diff
@@ -0,0 +1,52 @@
+import guid from './guid';
+
+describe('guid', () => {
+  beforeEach(() => {
+    Math.random = jest.fn(() => 0.4677647565236618);
```

Comment:
If this is always the case, you can move this outside of the `beforeEach`?

Reply:
done